insights/parsers/modinfo_i40e.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
How about putting all sub-class parsers in one single `modinfo.py` file?

Reply:
Ack, Done